Gemfile.lock

```diff
@@ -44,61 +44,60 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.3.1)
+    activeadmin (2.6.0)
       arbre (~> 1.2, >= 1.2.1)
       formtastic (~> 3.1)
       formtastic_i18n (~> 0.4)
       inherited_resources (~> 1.7)
       jquery-rails (~> 4.2)
       kaminari (~> 1.0, >= 1.0.1)
-      railties (>= 5.0, < 6.1)
+      railties (>= 5.2, < 6.1)
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-      sprockets-es6 (~> 0.9, >= 0.9.2)
-    activejob (6.0.0)
-      activesupport (= 6.0.0)
+    activejob (6.0.2.1)
+      activesupport (= 6.0.2.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activestorage (6.0.0)
-      actionpack (= 6.0.0)
-      activejob (= 6.0.0)
-      activerecord (= 6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activestorage (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activejob (= 6.0.2.1)
+      activerecord (= 6.0.2.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.0)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
+      zeitwerk (~> 2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.3.4)
+    airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.2)
+    annotate (3.0.3)
       activerecord (>= 3.2, < 7.0)
-      rake (>= 10.4, < 13.0)
+      rake (>= 10.4, < 14.0)
     apipie-rails (0.5.16)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
     aws-eventstream (1.0.3)
-    aws-partitions (1.219.0)
-    aws-sdk-core (3.68.0)
+    aws-partitions (1.258.0)
+    aws-sdk-core (3.86.0)
       aws-eventstream (~> 1.0, >= 1.0.2)
-      aws-partitions (~> 1.0)
+      aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.24.0)
-      aws-sdk-core (~> 3, >= 3.61.1)
+    aws-sdk-kms (1.27.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.24.0 → 1.27.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.24.0...1.27.0).

These are the first 10 commits:
- (0f7c40a) [Support S3 GET Object torrent](https://github.com/aws/aws-sdk-ruby/commit/0f7c40aa798e01dbb84f746f167d18e283cddee9)
- (527854b) [Fixed a bug with AutoScaling scheduled actions.](https://github.com/aws/aws-sdk-ruby/commit/527854bb2605f0008e5c0f10c9c8b28b8463143f)
- (93b7188) [Remove Unused Amazon ElastiCache Documentation](https://github.com/aws/aws-sdk-ruby/commit/93b718876f6ea2f020444e7fde50426649d8b200)
- (9fc9a88) [Fixed raise message](https://github.com/aws/aws-sdk-ruby/commit/9fc9a88cfaac42a6960503d349c5d69aa6256db7)
- (ad5b4df) [Remove Nokogiri < 1.6 restriction and explain in the README …](https://github.com/aws/aws-sdk-ruby/commit/ad5b4df099be3aaccea09232774e41a792e0b8fa)
- (8bfc473) [Merge pull request #407 from nishiyamaosamu/fix_s3_raise](https://github.com/aws/aws-sdk-ruby/commit/8bfc473ca49c0972eab59af4bdc92f811dd4d216)
- (5faa4b9) [Merge pull request #400 from grddev/support_s3_get_torrent](https://github.com/aws/aws-sdk-ruby/commit/5faa4b9b04acb1eaa3a710f8d0f8460510fe5730)
- (b1844f9) [Updating documentation about :max_retries to include informa…](https://github.com/aws/aws-sdk-ruby/commit/b1844f9b413bb217761dcc9f21ebbcb1d57de67b)
- (3a0e030) [New API Version 2013-06-30 for AWS Storage Gateway](https://github.com/aws/aws-sdk-ruby/commit/3a0e030bce80e5b5734a6db4740fd155f1f27663)
- (8dd20ce) [Add Cross-Zone Balancing to Elastic Load Balancing](https://github.com/aws/aws-sdk-ruby/commit/8dd20ce4d2bc1f6bd22f9120fd85db641a706a2d)


</details>